src/atoms/dataEntry/Radio/RadioItem.js

```diff
@@ -22,11 +27,14 @@ type RadioItemProps = {
   color?: 'primary' | 'secondary',
   /** when true then disable radio */
   disabled?: boolean,
-} & RadioItemClonedProps
+  /** custom function to check the equals */
+  equalsFunc?: (EqualsFuncArgument) => boolean,
```

Comment:
Where we should use it?

Reply:
Implement client-web feature. Used in the Settings/Security

Reply:
Why we should do this in UI kit, not app? In app we can use default radio and write some custom code for one case.